addons/binding/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/internal/kostal/inverter/ThirdGeneration/ThirdGenerationInverterTypes.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.internal.kostal.inverter.thirdgeneration;
+
+/**
+ * The {@link ThirdGenerationInverterTypes} contains the list of supported devices
+ * Each one is represented by a dedicated thing
+ *
+ * @author René Stakemeier - Initial contribution
+ */
+public enum ThirdGenerationInverterTypes {
+    PlenticorePlus42WithBattery,
```

Comment:
Enum should be upper case with preferable underscores between words.

Reply:
Damn Java :-)
Sorry, It´s a habbit from c# where CamelCase is the way to go… 
Fixing it in the next commit